library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -711,6 +711,14 @@ public void emitTuples()
   }
 
   /**
+   * Whether or not to advance the window now regardless of emitBatchSize.
+   */
+  protected boolean advanceWindow()
```

Comment:
Should it be suspendEmitTillNextWindow or something like that? I think suspendEmit can be confusing...